udfs/tests/dataform_testing_framework/deploy_and_run_tests.sh

```diff
@@ -240,12 +240,17 @@ main() {
     local udf_dirs
     # Get the list of directory names which contain UDFs
     udf_dirs=$(sed 's/:.*//g' <../../dir_to_dataset_map.yaml)
+    region_suffix=$(sed -rn "s/${BQ_LOCATION}: (.*)/\1/p" <../../region_to_dataset_suffix_map.yaml)
 
     for udf_dir in ${udf_dirs}; do
       # Get the short-hand version of the dataset_id
       # which is mapped in dir_to_dataset_map.yaml
       local dataset_id
       dataset_id=$(sed -rn "s/${udf_dir}: (.*)/\1/p" <../../dir_to_dataset_map.yaml)
+      # Region suffixes are used to deploy UDFs globally to bqutil without naming conflicts
+      if [[ "${PROJECT_ID}" = "bqutil" && "${BQ_LOCATION^^}" != "US" ]]; then
```

Comment:
```suggestion
      if [[  "${BQ_LOCATION^^}" != "US" ]]; then
```

Removing project check because we want to test region deployments in test environment